scripts/tsconfig/base.json

```diff
@@ -7,14 +7,16 @@
     "moduleResolution": "Node16",
     "declaration": true,
     "declarationMap": true,
-    "sourceMap": true,
+    "sourceMap": false,
     "isolatedModules": true,
     "strict": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "useDefineForClassFields": true,
     "rootDir": "${configDir}/src",
-    "outDir": "${configDir}/dist"
+    "outDir": "${configDir}/dist",
+    // TS reports TS5074 without this explicit setting
```

Comment:
This does not look like valid JSON

Reply:
Because it's not :trollface: TS is able to parse this fine though (they allow comments and trailing commas but I'm not 100% if they are using custom parsing rules or if they are compliant with json5 or jsonc)